testutil/keeper/application.go

```diff
@@ -38,6 +47,28 @@ func ApplicationKeeper(t testing.TB) (*keeper.Keeper, sdk.Context) {
 	mockBankKeeper.EXPECT().DelegateCoinsFromAccountToModule(gomock.Any(), gomock.Any(), types.ModuleName, gomock.Any()).AnyTimes()
 	mockBankKeeper.EXPECT().UndelegateCoinsFromModuleToAccount(gomock.Any(), types.ModuleName, gomock.Any(), gomock.Any()).AnyTimes()
 
+	mockAccountKeeper := mocks.NewMockAccountKeeper(ctrl)
+	mockAccountKeeper.EXPECT().GetAccount(gomock.Any(), gomock.Any()).AnyTimes()
+	// because the gateways are not staked this is needed to mock the GetPubKey method which
+	// returns nil if the actor cannot be found on chain (ie. not staked)
+	mockAccountKeeper.EXPECT().GetPubKey(gomock.Any(), gomock.Any()).DoAndReturn(func(_ sdk.Context, address sdk.AccAddress) (cryptotypes.PubKey, error) {
```

Comment:
Move `func(...)` to the next line and indent things appropriately for redability.

Ditto below.